sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/MiscellaneousOperationsLiveTests.cs

```diff
@@ -47,9 +47,16 @@ public async Task GetResourceDetails(bool useTokenCredential)
 
             ResourceQuotaDetails neuralQuota = resourceDetails.NeuralDocumentModelQuota;
 
-            Assert.GreaterOrEqual(neuralQuota.Used, 0);
-            Assert.GreaterOrEqual(neuralQuota.Quota, neuralQuota.Used);
-            Assert.Greater(neuralQuota.QuotaResetsOn, startTime);
+            if (_serviceVersion >= DocumentAnalysisClientOptions.ServiceVersion.V2023_07_31)
+            {
+                Assert.GreaterOrEqual(neuralQuota.Used, 0);
+                Assert.GreaterOrEqual(neuralQuota.Quota, neuralQuota.Used);
+                Assert.Greater(neuralQuota.QuotaResetsOn, startTime);
+            }
+            else
+            {
+                Assert.Null(neuralQuota);
+            }
```

Comment:
This is the expected behavior since the `NeuralDocumentModelQuota` property was introduced in service version 2023-07-31. Having the test working before means that the service was returning this property by mistake in an older API version, which seems to be fixed now.